ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -1456,7 +1457,7 @@ let migrate_send' ~__context ~vm ~dest ~live:_ ~vdi_map ~vif_map ~vgpu_map
             in
             inter_pool_metadata_transfer ~__context ~remote ~vm ~vdi_map
               ~vif_map ~vgpu_map ~dry_run:false ~live:true ~copy
-              ~check_cpu:(not force)
+              ~check_cpu:((not force) && power_state <> `Halted)
```

Comment:
how about 'Suspended' and 'Paused'? In theory you can move that around freely until you resume (at which point you do the CPU check).
Should the condition here we `power_state = 'Running` instead?

Reply:
This is an odd use case... The checks take into account that one may want to start/resume/unpause/shutdown the VM while it is being migrated to another pool. This is why the checks are always done with `~live:true`, which checks for available memory on the target host, even if the VM is halted. The CPU checks would be similar, except that you can't actually do it for a halted VM.